recipes/pyqode.qt/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
It looks like the `license_file` is already available in the source distribution so you can delete this file from the recipe.